### PR TITLE
I153 - remove survey data

### PR DESCRIPTION
**Pull request by hillalex**
Tests passing locally. 

(I'm trying to add `travis-ci` integration instead of `circle-ci` so we can stop ignoring these broken build warnings)

### CODE REVIEW

**Review by r-ash — CHANGES_REQUESTED** [review]
We should make the documentation and the expected column headers consistent.

At the moment order in the guidance text is
Survey Id, Country or region, Year, HIV Status, Sex, Age Group, Estimate, Standard Error, Lower Confidence Interval, Upper Confidence Interval, Counts

but order in the table headers and in the downloaded template is
Country or region, Survey Id, Year, Age Group, Sex, HIV Status, Estimate (%), Standard Error (%), Lower Confidence Interval (%), Upper Confidence Interval (%), Counts

If we update to use the latter we should make sure the survey data in sample_files is up to date.

File: src/server/survey_and_program_data.R [hunk 1]
```diff
@@ -151,16 +151,13 @@ surveyAndProgramData <- function(input, output, state, spectrumFilesState) {
         }"
 
     output$hot_survey <- rhandsontable::renderRHandsontable({
-        rhandsontable::rhandsontable(state$survey_data_human_readable(),
-        colHeaders = c("Country or region","Survey Id","Year","Age Group","Sex","HIV Status","Estimate (%)", "Standard Error (%)",
```
Comment (hillalex) [inline src/server/survey_and_program_data.R:164]: This `colHeaders` property is order-sensitive, so this was actually a bug - if users uploaded a CSV with a different ordering the model would work but the UI would be wrong! Have just removed this.

**Comment by hillalex** [comment]
Thanks, this led to me spotting a bug I hadn't noticed before - the displayed datatable had headers fixed in a specific order.

The app itself doesn't mind about header order as long as the headers are correctly named and the CSVs they upload may well have columns in a different order. 

But have updated the guidance text as suggested to reflect the initial ordering the page is populated with.

**Comment by r-ash** [comment]
Ah yes that was the behaviour I was seeing. Assumed that was a result of out of date sample data - didn't know about the app being column-order indifferent. My bad, should have explained what I was seeing instead of what I thought was the problem.

**Pull request merged**